.chloggen/semantic_source_identifier.yaml

```diff
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I think this can be considered a bug fix, since the migration was already done, but just one operator was missed.